posthog/temporal/tests/batch_exports/conftest.py

```diff
@@ -232,12 +232,16 @@ def data_interval_start(request, data_interval_end, interval):
 
 @pytest.fixture
 def data_interval_end(request, interval):
-    """Set a test data interval end."""
+    """Set a test data interval end.
+
+    This defaults to the current day at 00:00 UTC. This is done because event data is only available in events_recent
+    for the last 7 days, so if we try to insert data further in the past, it may be deleted and lead to flaky tests.
+    """
     try:
         return request.param
     except AttributeError:
         pass
-    return dt.datetime(2023, 4, 25, 15, 0, 0, tzinfo=dt.UTC)
+    return dt.datetime.now(tz=dt.UTC).replace(hour=0, minute=0, second=0, microsecond=0)
```

Comment:
Have you ran the bigquery test suite? We use freezegun and do a lot of `with freeze_time(TEST_TIME)` with a hardcoded `TEST_TIME`, so I'm wondering if this call is impacted in any way.